libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/signboard/api/MapboxSignboardApi.kt

```diff
@@ -23,9 +27,20 @@ import kotlinx.coroutines.launch
  */
```

Comment:
```suggestion
 * By default uses the [MapboxExternalFileResolver]. Check out [MapboxExternalFileResolver.resolveFont] to know how css style's fonts are treated.
 */
```

or something along these lines.